main/settings/src/main/scala/sbt/Remove.scala

```diff
@@ -0,0 +1,20 @@
+package sbt
+
+import scala.annotation.implicitNotFound
+
+object Remove {
+  @implicitNotFound(msg = "No implicit for Remove.Value[${A}, ${B}] found,\n  so ${B} cannot be remove from ${A}")
+  trait Value[A, B] extends Any {
+    def removeValue(a: A, b: B): A
+  }
+  @implicitNotFound(msg = "No implicit for Remove.Values[${A}, ${B}] found,\n  so ${B} cannot be remove from ${A}")
+  trait Values[A, -B] extends Any {
+    def removeValues(a: A, b: B): A
+  }
+  sealed trait Sequence[A, -B, T] extends Value[A, T] with Values[A, B]
+
+  implicit def removeSeq[T, V <: T] = new Sequence[Seq[T], Seq[V], V] {
```

Comment:
I beleive we have an `Equals` type class somewhere that we may want to eventually have here as a higher priority implicit.  For now, maybe we just ignore that.